### PR TITLE
48: vrp payment controller validations

### PR DESCRIPTION
**Pull request by jorgesanchezperez**
- Added new Errors codes for validations
Issue: https://github.com/OpenBankingToolkit/openbanking-toolkit/issues/48

### CODE REVIEW

File: src/main/java/uk/org/openbanking/datamodel/error/OBStandardErrorCodes1.java [hunk 1]
```diff
@@ -66,8 +66,7 @@
 
     UK_OBIE_INVALID_CONSENT_STATUS("UK.OBIE.Resource.InvalidConsentStatus"),
 
-    ;
-
+    UK_OBIE_FAILS_CONTROL_PARAMETERS("UK.OBIE.Rules.FailsControlParameters");
```
Comment (BohoCode) [inline src/main/java/uk/org/openbanking/datamodel/error/OBStandardErrorCodes1.java:69]: I'm sure I added this last week. Lemme find it!

Comment (BohoCode) [reply src/main/java/uk/org/openbanking/datamodel/error/OBStandardErrorCodes1.java:69]: Not sure what's going on here I added this Enum in this PR at line 50 ish
https://github.com/OpenBankingToolkit/openbanking-uk-datamodel/pull/140/files

Comment (BohoCode) [reply src/main/java/uk/org/openbanking/datamodel/error/OBStandardErrorCodes1.java:69]: This should be 
UK_OBIE_RULES_FAILS_CONTROL_PARAMETERS("UK.OBIE.Rules.FailsControlParameters"), shouldn't it?

Comment (jorgesanchezperez) [reply src/main/java/uk/org/openbanking/datamodel/error/OBStandardErrorCodes1.java]: Fixed!

**Review by BohoCode — APPROVED** [review]

**Pull request merged**